common_utilities/path_settings.py

```diff
@@ -14,7 +14,7 @@ class PathSettings:
     else:
         DOWNLOAD_PATH = Path('~/Downloads').expanduser()
 
-    NY_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir, os.pardir))
+    #NY_ROOT = os.path.abspath(os.path.join(BASE_DIR, os.pardir, os.pardir))
```

Comment:
you can remove this if not required

Reply:
Yeah it isn't -- [cb4e04943a84007ae233d1c08327345a3c4afa1b](https://github.com/dimagi/dimagi-qa/pull/145/commits/cb4e04943a84007ae233d1c08327345a3c4afa1b)